File: java-project/src/test/java/com/yourself/ExampleTest.java
package com.yourself;


import org.junit.Assert;
import org.junit.Test;

import java.io.IOException;


public class ExampleTest{

    @Test
    public  void test(){

        Assert.assertEquals("Did you return Hello World?", "Hello World!", Example.HelloWorld());
    }

    @Test
    public void visualisation() throws IOException {

        System.out.println("CG> open -s /project/answer/src/main/resources HelloWorld.html");

    }

}

File: kotlin-project/src/test/kotlin/com/yourself/ExampleTest.kt
package com.yourself

import org.junit.*
import java.io.*

class ExampleTest {
    @Test
    fun test() {
        Assert.assertEquals("Did you return Hello World?", "Hello World!", helloWorld())
    }

    @Test
    @Throws(IOException::class)
    fun visualisation() {
        println("CG> open -s /project/answer/src/main/resources HelloWorld.html")
    }
}